### PR TITLE
Fix tests following convention's project naming

### LINKED ISSUE #425: AbstractBlockServiceTestCase should be renamed to BlockServiceTestCase
Test case are should not prefixed by the `Abstract` key, like this: https://github.com/symfony/symfony/blob/ba3b17719272e614732a3d1323b556cf32da8dfb/src/Symfony/Component/Form/Test/TypeTestCase.php#L18

As it's already a moved, we may deprecate this one after 4.0 release.

### PR DESCRIPTION
**Pull request by Th3Mouk**
I am targeting this branch, because it don't break anything.

Closes #425

## Changelog

```markdown

### Changed
- 1 class name in tests

```

### CODE REVIEW

File: src/Test/BlockServiceTestCase.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
  *
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
-abstract class AbstractBlockServiceTestCase extends TestCase
+abstract class BlockServiceTestCase extends TestCase
```
Comment (kunicmarko20) [inline src/Test/BlockServiceTestCase.php]: This is a BC Break, this file is in `src`

Comment (core23) [reply src/Test/BlockServiceTestCase.php]: You should introduce a wrapper class and deprecate `AbstractBlockServiceTestCase`

**Comment by OskarStark** [comment]
Any news here?

**Comment by OskarStark** [comment]
Lets close it then.

Feel free to reopen any time 👌🏻

**Pull request closed without merging**